### PR TITLE
Selects & Ghost Fields

### PR DESCRIPTION
**Pull request by ludwiczakpawel**
## What type of PR is this? (check all applicable)

- [x] Feature

## Description

This PR does two things:

1. It brings "ghost" styling for form textfields. It basically gets rid of ANY decorations and visuals - it's just an editable box. 
2. It brings styling for `<select>` elements.

Both will be utilized for "write a post" new design (in fact, it's already used but I'm trying to make my other PR smaller :D).

## Screenshots

image

## Related Tickets & Documents

- #7524 

## Added tests?

- [x] no, because they aren't needed

## Added to documentation?

- [x] no documentation needed

### CODE REVIEW

**Comment by rhymes** [comment]
Can you provide a screenshot :D ? As I'm not sure how to visualize this :D

**Comment by Ridhwana** [comment]
Hmm same here: it looks like there should have been a classname `crayons-select` to test with on the user misc page but it's not showing up on the element so its difficult to see the effects of the css.

```
    <div class="crayons-field">
      <%= f.label :inbox_type, "Inbox type", class: "crayons-field__label" %>
      <p class="crayons-field__description">Open your inbox to messages from people you don't follow or keep your inbox private to mutual follows.</p>
      <%= f.select :inbox_type, [%w[Open open], %w[Private private]], class: "crayons-select" %>
    </div>
```

Screenshot 2020-04-30 at 11 23 45

File: app/assets/stylesheets/components/forms.scss [hunk 1]
```diff
@@ -183,6 +219,18 @@
   }
 }
 
+///////////////////////////////////////////////////
+.crayons-textfield {
```
Comment (Ridhwana) [inline app/assets/stylesheets/components/forms.scss:223]: Is a text field an input field or a text area?

Comment (ludwiczakpawel) [reply app/assets/stylesheets/components/forms.scss:223]: Correct, you can use `crayons-textfield` for both: `<input type="text">` and `<textarea>`

**Comment by ludwiczakpawel** [comment]
@rhymes @Ridhwana I've added screenshot to the PR description.

>  it looks like there should have been a classname crayons-select to test with on the user misc page but it's not showing up on the element so it's difficult to see the effects of the css.

I believe there was some syntax error. Even though it wasn't part of this PR, I've fixed it here since it's a quick fix.

File: app/views/pages/crayons.html.erb [hunk 2]
```diff
@@ -277,22 +277,54 @@
...
     <textarea class="crayons-textfield" placeholder="This is palceholder..."></textarea><br><br>
-    <textarea class="crayons-textfield" placeholder="This is palceholder..." disabled>Disabled textarea</textarea>
+    <textarea class="crayons-textfield" placeholder="This is palceholder..." disabled>Disabled textarea</textarea><br><br>
+    <textarea class="crayons-textfield crayons-textfield--ghost" placeholder="This is palceholder for ghost textarea..."></textarea>
   </div>
 
   <pre><%= '
   <textarea class="crayons-textfield" placeholder="This is palceholder..."></textarea>
   <textarea class="crayons-textfield" placeholder="This is palceholder..." disabled>Disabled textarea</textarea>
+  <textarea class="crayons-textfield crayons-textfield--ghost" placeholder="This is palceholder for ghost textarea..."></textarea>
```
Comment (Ridhwana) [inline app/views/pages/crayons.html.erb:299]:
```suggestion
  <textarea class="crayons-textfield crayons-textfield--ghost" placeholder="This is placeholder for ghost textarea..."></textarea>
```

small typo 😉

File: app/views/pages/crayons.html.erb [hunk 3]
```diff
@@ -277,22 +277,54 @@
...
   <div>
     <textarea class="crayons-textfield" placeholder="This is palceholder..."></textarea><br><br>
-    <textarea class="crayons-textfield" placeholder="This is palceholder..." disabled>Disabled textarea</textarea>
+    <textarea class="crayons-textfield" placeholder="This is palceholder..." disabled>Disabled textarea</textarea><br><br>
+    <textarea class="crayons-textfield crayons-textfield--ghost" placeholder="This is palceholder for ghost textarea..."></textarea>
   </div>
 
   <pre><%= '
   <textarea class="crayons-textfield" placeholder="This is palceholder..."></textarea>
   <textarea class="crayons-textfield" placeholder="This is palceholder..." disabled>Disabled textarea</textarea>
```
Comment (Ridhwana) [inline app/views/pages/crayons.html.erb:298]: LOL copy and paste typos in this file on the placeholder key ```placeholder="This is palceholder...">```

File: app/views/pages/crayons.html.erb (hunk 2 above)
Comment (ludwiczakpawel) [reply app/views/pages/crayons.html.erb]: thanks, fixed!

File: app/views/pages/crayons.html.erb (hunk 3 above)
Comment (ludwiczakpawel) [reply app/views/pages/crayons.html.erb]: :)

**Pull request merged**